apis/v1alpha1/httproute_types.go

```diff
@@ -274,14 +274,16 @@ type HTTPRouteMatch struct {
 }
 
 const (
-	// FilterHTTPReqeustHeader can be used to add or remove an HTTP
+	// FilterTypeHTTPRequestHeader can be used to add or remove an HTTP
 	// header from an HTTP request before it is sent to the upstream target.
 	// Support: core
 	// +optional
-	FilterHTTPRequesttHeader = "RequestHeader"
+	FilterTypeHTTPRequestHeader = "RequestHeader"
 	// TODO(hbagdi): add FilterHTTPRequestMirroring filter
 
-	FilterImplementationSpecific = "ImplementationSpecific"
+	// FilterTypeImplementationSpecific should be used for configuring
+	// custom filters.
+	FilterTypeImplementationSpecific = "ImplementationSpecific"
```

Comment:
Just to clarify; this constant can be used with to filter other route types?

Reply:
Yes.